browser/scripts/abstractPlugins/abstractObjectGazePlugin.js

```diff
@@ -9,7 +9,7 @@
 				return
 
 			subgraph.nodes.map(function(node) {
-				if (node.plugin.id === 'graph')
+				if (E2.GRAPH_NODES.indexOf(node.plugin.id) > -1)
```

Comment:
found this pattern everywhere but over here specifically a plugin knows something about E2.GRAPH_NODES. justsaying..